src/main/plugin/iso19139.ca.HNAP/formatter/xsl-view/view.xsl

```diff
@@ -52,8 +53,10 @@
   <xsl:variable name="editorConfig"
                 select="document('../../layout/config-editor.xml')"/>
 
-  
-  <!-- Override codelist template, to don't use the text value, 
+  <xsl:variable name="langId" select="gn-fn-iso19139:getLangIdHNAP($metadata, $language)"/>
```

Comment:
Would it not be better to use something like the following instead of creating a new gn-fn-iso19139:getLangIdHNAP function?

`gn-fn-iso19139:getLangId($metadata, xslUtils:iso639_2B_to_iso639_2T($language))`